test/utils/test.mjs

```diff
@@ -8,19 +8,23 @@ import defaultParserOptions from './default-parser-options.mjs';
 
 const require = createRequire(import.meta.url);
 
-function normalizeInvalidTest(test, rule) {
-	const {code, output, errors} = test;
+function verifySuggestionPropertyConsistency(rule, invalidTests) {
+	for (const invalidTest of invalidTests) {
+		if (Array.isArray(invalidTest.errors) && invalidTest.errors.some(error => error.suggestions) && !(rule.meta && rule.meta.docs && rule.meta.docs.suggestion === true)) {
```

Comment:
Let's use `rule.meta.docs.suggestion` directly, because `rule.meta.docs` is required field.

Reply:
Updated.